docs/content/chunker.md

```diff
@@ -299,6 +299,9 @@ If wrapped remote is case insensitive, the chunker overlay will inherit
 that property (so you can't have a file called "Hello.doc" and "hello.doc"
 in the same directory).
 
+Chunker included in rclone releases up to `v1.53.2` can sometimes fail to
```

Comment:
This should probably say "Chunker included in rclone releases prior to `v1.54.0` can..."

Note that we've released v1.53.3 already! I think we should target this for v1.54.0

Reply:
Yes, I intended the sequence of commits to be:
- merge it on master for v1.54.0
- cherry pick from master to v.1.53-stable for the next point release
- rebase PR 4675 on new master

Reply:
Do you mean that 1.53.4 release would be close to 1.54, making backport moot?